lib/saml_config.rb

```diff
@@ -2,20 +2,18 @@
 
 module Saml
   class Config
-    # rubocop:disable AbcSize, MethodLength, CyclomaticComplexity, PerceivedComplexity
+    # rubocop:disable AbcSize, MethodLength
     def self.build_settings
-      config_file = "#{Rails.root}/config/saml.yml"
-      saml_env_config = Hashie::Mash.new(YAML.load_file(config_file).fetch(Rails.env, {}))
       Hashie::Mash.new(
-        issuer: Figaro.env.SP_ISSUER || saml_env_config[:sp_issuer],
-        idp_sso_target_url: Figaro.env.IDP_SSO_URL || saml_env_config[:sso_url],
-        idp_slo_target_url: Figaro.env.IDP_SLO_URL || saml_env_config[:slo_url],
-        idp_cert_fingerprint: Figaro.env.IDP_FINERPRINT || saml_env_config[:idp_fingerprint],
-        name_identifier_format: Figaro.env.IDP_NAME_ID_FORMAT || saml_env_config[:name_id_format],
-        certificate: Figaro.env.SP_CERTIFICATE || saml_env_config[:sp_certificate],
+        issuer:                     Figaro.env.saml_sp_issuer,
```

Comment:
we haven't been using this style of lining up vals, but no biggie, can clean up l8r